config/global.go

```diff
@@ -111,6 +111,44 @@ func GetDaemonProfileCPUDuration() int64 {
 	return globalConfig.origin.SystemControllerConfig.DebugConfig.ProfileDuration
 }
 
+func GetTarfsExportEnabled() bool {
+	switch globalConfig.origin.Experimental.TarfsConfig.ExportMode {
+	case "layer_verity_only":
+		return true
+	case "image_verity_only":
+		return true
+	case "layer_block":
+		return true
+	case "image_block":
+		return true
+	case "layer_block_with_verity":
+		return true
+	case "image_block_with_verity":
+		return true
+	default:
+		return false
+	}
+}
+
+func GetTarfsExportFlags() (bool, bool, bool) {
```

Comment:
It's better to have a comment describe the multiple return values, their order is crucial

Reply:
Good point, add a comment to describe the return values.